extension/js/common/composer.ts

```diff
@@ -1201,6 +1208,45 @@ export class Composer {
         if (method === 'forward') {
           this.urlParams.subject = 'Fwd: ' + this.urlParams.subject;
           await this.retrieveDecryptAddForwardedMsg(determined.lastMsgId);
+        } else {
+          try {
+            const { raw } = await Google.gmail.msgGet(this.urlParams.acctEmail, this.urlParams.threadMsgId, 'raw');
+            const msgMimeContent = await Mime.process(Buf.fromBase64UrlStr(raw!));
+            if (!msgMimeContent.from || !msgMimeContent.headers.date || !msgMimeContent.blocks || !msgMimeContent.blocks.length) {
+              return;
```

Comment:
This is a fair precaution, though I cannot imagine it would actually happen. Maybe indeed `!msgMimeContent.blocks.length` could be true for empty messages, not sure.

But it's ok to check, just some thoughts.